charter/charter.md

```diff
@@ -18,7 +18,7 @@ QUIC specifications:
 - Maintenance and evolution of the existing QUIC extensions specified
   by the WG.
 
-Maintenance and evolution work needs to be strongly motivated by
+The working group adoption of maintenance and evolution work for QUIC base specifications needs to be strongly motivated by
```

Comment:
```suggestion
WG adoption of maintenance and evolution work for the QUIC base specifications needs to be strongly motivated by
```

Reply:
modifications LGTM. thanks.

Reply:
I think we want apply the same criteria to the base drafts or any adopted extensions.

Reply:
if these are two cases to cover then I would rather add "or any adopted extensions" to the sentence. This will clarify the scope of the intention here.